cmd/experimental/kueue-viz/frontend/src/Cohorts.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { Typography, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, CircularProgress } from '@mui/material';
+import useWebSocket from './useWebSocket';
+import './App.css';
+
+const Cohorts = () => {
+  const { data: cohorts, error } = useWebSocket('/ws/cohorts');
+  const [cohortList, setCohortList] = useState([]);
+
+  useEffect(() => {
+    if (cohorts && Array.isArray(cohorts)) {
+      setCohortList(cohorts);
+    }
+  }, [cohorts]);
+
+  if (error) return <Typography color="error">{error}</Typography>;
+
+  return (
+    <Paper style={{ padding: '16px', marginTop: '20px' }}>
+      <Typography variant="h4" gutterBottom>Cohorts</Typography>
+      {cohortList.length === 0 ? (
```

Comment:
If I don't have cohorts, it loads indefinitely.

Reply:
Also other pages.

Reply:
Initially, the loading of list was taking quite long , and I had this circular waiting icon.
But, it does not make sense anymore.
I removed it and added an explicit message.
However, now, the message appears just before the list loads.
Let's have this temporarly, and then we can improve user experience by having a loading state that can get trigger the message properly if no results are found.